GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -698,8 +705,14 @@ private void writeAnswer(
 
         int col = startColumn;
         for (String cellValue : cells) {
-            if (questionType == QuestionType.NUMBER) {
+        	if (questionType == QuestionType.NUMBER) {
```

Comment:
Of course. ( I made a new Eclipse workspace just for Flow and got vanilla editor settings...)